src/lib.rs

```diff
@@ -591,14 +637,23 @@ macro_rules! create_app {
                             .long("enclave-id")
                             .takes_value(true)
                             .help("Enclave ID, used to uniquely identify an enclave")
-                            .required(true),
+                            .required(true)
```

Comment:
You are looking for `required_unless("enclave-name")` which requires an argument unless other argument `enclave-name` is found